docker-compose.yml

```diff
@@ -12,7 +12,14 @@ services:
       - NVIDIA_VISIBLE_DEVICES=all
     volumes:
       - .:/workspace
-  
+    deploy:
```

Comment:
don't know how to make this optional

Reply:
is this `deploy` block mainly used in docker swarm? Not sure if it impacts github actions at all

Reply:
Unsure about github actions impact.

No luck w/ `runtime` option, unless I didn't do it right. I tested by hardcoding `runtime: nvidia` and got the following error:

```
Creating cvlization_app_run ... error

ERROR: for cvlization_app_run  Cannot create container for service app: Unknown runtime specified nvidia

ERROR: for app  Cannot create container for service app: Unknown runtime specified nvidia
ERROR: Encountered errors while bringing up the project.
```